.github/workflows/auto-tag-release.yaml

```diff
@@ -0,0 +1,29 @@
+name: Update Semver Tags
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+      
+permissions:
+  contents: write
+
+jobs:
+  update-semver-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
+      
+      - name: Setup node
+        uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d #v3.8.1
```

Comment:
Is this action/node necessary here?

Reply:
This action is a exact copy from dbp-github-workflows, I will check if this is needed.

Reply:
action node is not needed and removed.